packages/config/src/layer2s/optimism.ts

```diff
@@ -34,6 +34,7 @@ export const optimism: Layer2 = {
   display: {
     name: 'Optimism',
     slug: 'optimism',
+    headerWarning: 'Optimism is currently undergoing a major upgrade.',
```

Comment:
@lucadonnoh Can you please do the wording here?

Reply:
i'd like to add a link to the optimism website page that contains additional info, is it possible?